network/p2p/libp2pNode.go

```diff
@@ -478,6 +483,19 @@ func (n *Node) Subscribe(ctx context.Context, topic flownet.Topic) (*pubsub.Subs
 		if err != nil {
 			return nil, fmt.Errorf("could not join topic (%s): %w", topic, err)
 		}
+
+		if len(validators) > 0 {
+			if err := n.pubSub.RegisterTopicValidator(
```

Comment:
Can we register before we do a `n.pubSub.Join` ?

Reply:
We can, sure.